oleesir/authors-haven-api
# Logout user

**What does this Merge request do?**

Implements feat__loginUser

**Description of Tasks completed?**

Implements feat__feat__loginUser

**How should this be manually tested?**

- git fetch on your local machine and check out to feat__loginUser branch
- Run "npm run dev" to start the development server

**Any background context you want to provide?**

N/A


**Screenshots (if appropriate)**
